src/symbols.js

```diff
@@ -746,10 +739,6 @@ defineSymbol(text, main, mathord, "\u00a3", "\\textsterling", true);
 defineSymbol(math, ams, textord, "\u2720", "\\maltese");
 defineSymbol(text, ams, textord, "\u2720", "\\maltese");
 
-defineSymbol(text, main, spacing, "\u00a0", "\\ ");
-defineSymbol(text, main, spacing, "\u00a0", " ");
-defineSymbol(text, main, spacing, "\u00a0", "~");
```

Comment:
These are dupes, right?

Reply:
@kevinbarabash Yes, they have duplicates in:
https://github.com/KaTeX/KaTeX/blob/0d8830af304c722814df6060a170faa5d94d0592/src/symbols.js#L592-L594